packages/foodie-backend/src/app.ts

```diff
@@ -17,19 +18,21 @@ app.use(helmet());
 app.use(cors());
 app.set('trust proxy', 1); // trust first proxy
 
-app.use('/api/v1', [AuthRouter, UserRouter]);
+app.use('/api/v1', [AuthRouter, UserRouter, VendorRouter]);
```

Comment:
I will suggest, we define one en

Reply:
like having an index.js to bundle all routes yh ?
would do that